frontend/src/components/FileUploadForm.vue

```diff
@@ -135,7 +137,8 @@
                 uploadIndex: 0,
                 fileIds: [],
                 spanKey: 0,
-                readyToUpload: false,
+                readyToUpload: true,
+                uploading: false,
```

Comment:
In the code above there's a JS object that didn't end w/ comma. Maybe use a linter and apply the format consistently?